index.html

```diff
@@ -40,252 +46,272 @@
+              <li>Month YYYY: First face-to-face meeting</li>
+              <li>Month YYYY: Requirements and Use Cases for FooML</li>
+              <li>Month YYYY: FPWD for FooML</li>
+              <li>Month YYYY: Requirements and Use Cases for BarML</li>
+              <li>Month YYYY: FPWD FooML Primer</li>
+            </ul>
+        </section>
+      </section>
 
-      <div id="ig-other-deliverables">
-        <h3>Other deliverables</h3>
+	<section id="success-criteria">
+	  <h2>Success Criteria</h2>
+	  <p>In order to advance to <a href="https://www.w3.org/Consortium/Process/#RecsPR" title="Proposed Recommendation">Proposed Recommendation</a>, each normative specification is expected to have <a href="https://www.w3.org/Consortium/Process/#implementation-experience">at least two independent implementations</a> of every feature defined in the specification.</p>
 
-        <p>The Working Group MAY create documents that are not Technical Reports, including:</p>
+    <p>Each Recommendation-track Technical Report:</p>
 
-        <ul>
-          <li>Use case and requirement documents</li>
+    <ul>
+      <li>SHOULD contain a section detailing all known security and privacy implications for implementers, Web authors, and end users.</li>
+      <li>SHOULD have an associated testing plan, starting from the earliest drafts.</li>
+      <li>SHOULD contain a section on accessibility that describes the benefits and impacts, including ways that features defined in the Technical Report can be used to address them, and recommendations for maximising accessibility in implementations.</li>
+      <li>SHOULD address the <a href="https://www.w3.org/TR/media-accessibility-reqs/">Media Accessibility User Requirements</a></li>
+      <li>SHOULD contain a section discussing interoperability with previous versions, if any, of the Technical Report, and other relevant specifications.</li>
+    </ul>
+    <p>To promote interoperability, all normative changes made to Technical Reports SHOULD have associated <a href='https://www.w3.org/2019/02/testing-policy.html'>tests</a>.</p>
 
-          <li>Test suite and implementation reports</li>
+<!-- need to consider of -->
+    <p><i class="todo">NEED TO SELECT AND CONSIDER OF FOLLOWING ITEMS IN THE TEMPLATE.</i></p>
+    <p>Each specification should contain separate sections detailing all known security and privacy implications for implementers, Web authors, and end users.</p>
+	  <p>There should be testing plans for each specification, starting from the earliest drafts.</p>
+	  <p><i class="todo">For specifications of technologies that directly impact user experience, such as content technologies, as well as protocols and APIs which impact content: </i>
+		Each specification should contain a section on accessibility that describes the benefits and impacts, including ways specification features can be used to address them, and
+		recommendations for maximising accessibility in implementations.</p>
```

Comment:
Should we have a separate section for this, or should it be part of the specs we have in general? Since captions are in themselves accessibility features.

Reply:
text says `a section on accessibility`, but not `separate sections detailing ...` like a line above...

Reply:
Sorry, I don't understand the question @gkatsev - or your answer @himorin !